MekHQ/src/mekhq/campaign/parts/ProtomekArmActuator.java

```diff
@@ -85,7 +84,7 @@ public Money getStickerPrice() {
     @Override
     public boolean isSamePartType (Part part) {
         return part instanceof ProtomekArmActuator
-                && getUnitTonnage() == ((ProtomekArmActuator)part).getUnitTonnage();
+                && getUnitTonnage() == part.getUnitTonnage();
```

Comment:
Double check that ProtomekArmActuator doesn't have an implementation of getUnitTonnage()